Mark NewVolumeManagerReconstruction as GA (#45282)
kubernetes/website: content/en/docs/reference/command-line-tools-reference/feature-gates/new-volume-manager-reconstruction.md

  - stage: beta
    defaultValue: true
    fromVersion: "1.28"
    toVersion: "1.29"
  - stage: stable
    defaultValue: true
    fromVersion: "1.30"
---
Enables improved discovery of mounted volumes during kubelet

> startup. Since this code has been significantly refactored, we allow to opt-out in case kubelet

Author: I rephrased the text + updated Kubernetes versions where the feature was alpha / beta. It should be final version for review, I'm removing `WIP:`.

  - stage: beta
    defaultValue: true
    fromVersion: "1.28"
    toVersion: "1.29"
  - stage: stable
    defaultValue: true
    fromVersion: "1.30"
---
Enables improved discovery of mounted volumes during kubelet

> startup. Since this code has been significantly refactored, Kubernetes versions 1.25 - 1.29
> allowed to opt-out in case kubelet gets stuck at the startup or is not unmounting volumes
> from terminated Pods. Note that this refactoring was behind `SELinuxMountReadWriteOncePod`
> alpha feature gate in versions 1.25 - 1.26.

Reviewer: ```suggestion
startup. Since the associated code had been significantly refactored, Kubernetes versions 1.25 to 1.29
allowed you to opt-out in case the kubelet got stuck at the startup, or did not unmount volumes
from terminated Pods.

This refactoring was behind the `SELinuxMountReadWriteOncePod`  feature gate in Kubernetes
releases 1.25 and 1.26.
```
